kuksa-client/tests/test_grpc.py

```diff
@@ -1313,6 +1313,51 @@ async def test_set_some_updates_v2(
             ):
                 assert actual_request == expected_request
 
+    @pytest.mark.usefixtures("val_server")
```

Comment:
rename to databroker?

Reply:
I first thought it was a good idea, but then I saw that the term `val_server` occurs quite frequent in generated code, and the service databroker provides is actually called `service VAL` so the name sorts of make sense.

Possibly `service Databroker` would have been a better name and maybe that we should have used that name for v2, but oo late now I suppose .

https://github.com/eclipse-kuksa/kuksa-databroker/blob/main/proto/kuksa/val/v2/val.proto#L23
https://github.com/eclipse-kuksa/kuksa-databroker/blob/bfcb1e480833c59925ca6f57c45df02e41935982/databroker/src/grpc/kuksa_val_v1/val.rs#L41